twake/frontend/src/app/molecules/media/index.tsx

```diff
@@ -0,0 +1,49 @@
+import React from 'react';
+
+// @ts-ignore
+
+interface MediaProps extends React.InputHTMLAttributes<HTMLInputElement> {
+  size: MediaSize;
+  url: string;
+  duration?: string;
+}
+
+export enum MediaSize {
+  LG = 32,
+  MD = 16,
+  SM = 12,
+}
```

Comment:
On other components I put "sm" "md" "lg", could we use the same ? (I don't mind having a enum that's ok)

Reply:
So are you want me to move to strings, or just me to create the universal Enum for all the atoms/molecules?

Reply:
Move to string